src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -25,27 +32,51 @@ const onDownloadClick = url => event => {
   }
 };
 
+// Helper to derive the download link props.
+const deriveLinkProps = form => {
+  const linkProps = {};
+
+  const isSameOrigin = isURLSameOrigin(form?.attributes?.url);
+  const isPDF = form?.attributes?.url.toLowerCase().includes('.pdf');
+  const isInternetExplorer = !!navigator.msSaveBlob;
+
+  if (!isSameOrigin || !isPDF) {
+    // Just open in a new tab if we'd otherwise hit a CORS issue or if the form URL isn't a PDF.
+    linkProps.target = '_blank';
+  } else if (isInternetExplorer) {
+    // IE doesn't support the HTML download attribute.
+    linkProps.onClick = onDownloadClick(form.attributes.url);
+  } else {
+    // Use HTML5 `download` attribute.
+    linkProps.download = true;
+  }
+
+  return linkProps;
+};
+
 const SearchResult = ({ form }) => {
+  // Escape early if we don't have the necessary form attributes.
   if (!form?.attributes) {
     return null;
   }
 
-  const pdf = form.attributes.url.toLowerCase().includes('.pdf') ? '(PDF)' : '';
+  // Derive the download link props.
+  const linkProps = deriveLinkProps(form);
+
+  // Derive labels.
+  const pdfLabel = form.attributes.url.toLowerCase().includes('.pdf')
+    ? '(PDF)'
+    : '';
   const lastRevisionOn = form.attributes.lastRevisionOn
     ? moment(form.attributes.lastRevisionOn).format('MM-DD-YYYY')
     : 'N/A';
 
-  const formTitleClassName = [
-    'vads-u-padding-top--3',
-    'vads-u-margin--0',
-    'vads-u-border-top--1px',
-    'vads-u-border-color--gray-lighter',
-    'vads-u-font-weight--bold',
-  ].join(' ');
-
   return (
     <>
-      <dt data-e2e-id="result-title" className={formTitleClassName}>
+      <dt
+        className="vads-u-padding-top--3 vads-u-margin--0 vads-u-border-top--1px vads-u-border-color--gray-lighter vads-u-font-weight--bold"
```

Comment:
I just inlined this since we're not rendering any conditional classes here, let me know if you'd prefer I revert that though! 🙂